### DISCUSSION TITLE
How to manually stop streams?

### DISCUSSION THREAD

**Discussion by LaAlexe** [yume-chan/ya-webadb#614] [Q&A] [answered]
I found the comments in the demo of library :
// Adb won't close the streams,
// so manually close them.
But I didn`t get how to do it?

**Comment by yume-chan** [comment]
Where are those comments?

**Reply by LaAlexe** [reply]
there was an old-demo project, I downloaded it, and it is in packages/demo/src/components/connect.tsx file in dispose() method

**Reply by yume-chan** [reply, marked as answer]
The following lines are how to stop streams.

https://github.com/tango-adb/old-demo/blob/74e9154e5e474812b9e9424b89e5d962d0c9d174/packages/demo/src/components/connect.tsx#L219-L226
